content/docs/self-hosted-runners.md

```diff
@@ -1,82 +1,81 @@
-# Self-hosted Runners
+# Self-hosted (On-premise or Cloud) Runners
 
-GitHub Actions and GitLab CI are run on GitHub- and GitLab- hosted runners by
-default. However, there are many great reasons to use your own runners: to take
-advantage of GPUs; to orchestrate your team's shared computing resources, or to
-train in the cloud.
+GitHub Actions and GitLab CI workflows are run on GitHub- and GitLab- hosted
+runners by default. However, there are many great reasons to use your own
+runners: to take advantage of GPUs; to orchestrate your team's shared computing
+resources, or to train in the cloud.
 
 ☝️ **Tip!** Check out the official documentation from
 [GitHub](https://help.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners)
-and [GitLab](https://docs.gitlab.com/runner/) to get started setting up your
+and [GitLab](https://docs.gitlab.com/runner) to get started setting up your
 self-hosted runner.
 
-## Allocating cloud resources with CML
+## Allocating Cloud Compute Resources with CML
 
-When a workflow requires computational resources (such as GPUs) CML can
+When a workflow requires computational resources (such as GPUs), CML can
 automatically allocate cloud instances using `cml-runner`. You can spin up
-instances on your AWS or Azure account (GCP support is forthcoming!).
+instances on your [AWS](#aws), [Azure](#azure), [GCP](#gcp), or
+[Kubernetes](#kubernetes) accounts.
```

Comment:
```suggestion
instances on [AWS](#aws), [Azure](#azure), [GCP](#gcp), or
[Kubernetes](#kubernetes).
```

Not sure if “account” is the most appropriate term to describe where you can spin up instances. What if we obliterate it?